doc/code_of_conduct_enforcement_manual.rst

```diff
@@ -0,0 +1,60 @@
+.. _Zephyr_Code_of_Conduct_Enforcement_Manual
+
+Code of Conduct Enforcement Manual
+**********************************
+
+This is the enforcement manual followed by the Zephyr Projects’ Code of Conduct Working Group. It's used when we respond to an issue to make sure we're consistent and fair. It should be considered an internal document, but we're publishing it publicly in the interests of transparency.
+
+The Code of Conduct Working Group
+*********************************
+
+All responses to reports of conduct violations will be managed by a Code of Conduct Working Group ("the CoC WG").
+The Zephyr Project’s Governing Board ("the board") will establish this working group, comprised of at least five members. One member will be designated chair of the group and will be responsible for all reports back to the board. The board will review membership of the CoC WG on an annual basis. The CoC WG will consist of the project leadership (Governing Board Chair, TSC chair, Marketing Committee Chair, Community Representative and Project Director). 
+
+How the Code of Conduct Working Group Will Respond to Reports
+*************************************************************
+
+When a report is sent to the committee they will immediately reply to the report to confirm receipt. The reporting email address is **report@zephyrproject.org**. This reply must be sent within 24 hours, and the group should strive to respond much quicker than that.
+
+See the reporting guidelines for details of what reports should contain. If a report doesn't contain enough information, the CoC WG will obtain all relevant data before acting. The CoC WG is empowered to act on the Zephyr Project’s behalf in contacting any individuals involved to get a more complete account of events.
+
+The CoC WG will then review the incident and determine, to the best of their ability:
+ * What happened
```

Comment:
need a blank line before a list, and don't indent the items:

    The CoC WG will then review the incident and determine, to the best of their ability:

    * What happened